framework/src/loops/ComputeMaterialsObjectThread.C

```diff
@@ -153,6 +153,96 @@ ComputeMaterialsObjectThread::onBoundary(const Elem * elem, unsigned int side, B
   }
 }
 
+void
+ComputeMaterialsObjectThread::onInterface(const Elem * elem, unsigned int side, BoundaryID bnd_id)
+{
+  if (_fe_problem.needBoundaryMaterialOnInterafce(bnd_id, _tid))
+  {
+    _assembly[_tid]->reinit(elem, side);
+    unsigned int face_n_points = _assembly[_tid]->qRuleFace()->n_points();
+    _bnd_material_data[_tid]->resize(face_n_points);
+    _neighbor_material_data[_tid]->resize(face_n_points);
+
+    if (_has_bnd_stateful_props)
+    {
+      // Face Materials
+      if (_discrete_materials[Moose::FACE_MATERIAL_DATA].hasActiveBlockObjects(_subdomain, _tid))
+      {
```

Comment:
Executing a single statement; you don't need the `{`. Same for other places in this method